cmd/columnify/columnify.go

```diff
@@ -18,6 +18,16 @@ func printUsage() {
 	flag.PrintDefaults()
 }
 
+func columnify(c columnifier.Columnifier, files []string) (err error) {
+	defer func() {
+		err = c.Close()
```

Comment:
This assignment overwrites the `err` value, even if `c.WriteFromFiles(files)` returns non-nil error. So the error from `c.WriteFromFiles(files)` is not reported anymore. The correct fix is, call `c.Close()`, but assign to `err` only when the value (returned by `c.WriteFromFiles(files)` is `nil`.

Reply:
oops, thanks for your suggestion! I've just encountered this issue ... anyway I created an issue for that https://github.com/reproio/columnify/issues/49 and will fix it.